mono/mini/aot-compiler.c

```diff
@@ -4861,7 +4861,8 @@ add_generic_class_with_depth (MonoAotCompile *acfg, MonoClass *klass, int depth,
 			if (strstr (method->name, name_prefix)) {
```

Comment:
Should this be strncmp? strstr is much slower, and the term `prefix` here suggests strstr is overkill.